### PR TITLE
update Trail text

### PR DESCRIPTION
**Pull request by aneelac22**
[RHCLOUD-20834]
Ref: https://docs.google.com/document/d/1X3cmbsDO8FvPh5wu900lwLSUefYkhfEpgEiCteNlwqI/edit

### CODE REVIEW

File: src/messages/trial.messages.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ const trialMessages = defineMessages({
   },
   startInstallTrial: {
     id: 'trial.startInstallTrial',
-    defaultMessage: 'You may need to wait up to 30 minutes to access <a>automation hub</a> services after the trial subscription is activated.'
+    defaultMessage: 'You may need to wait up to 30 minutes to access <a href="https://cloud.redhat.com/ansible/automation-hub">automation hub</a> services after the trial subscription is activated.'
```
Comment (karelhala) [inline src/messages/trial.messages.js:168]: cloud URLs are automatically redirected to console ones. Let's use that here directly
```suggestion
    defaultMessage: 'You may need to wait up to 30 minutes to access <a href="https://console.redhat.com/ansible/automation-hub">automation hub</a> services after the trial subscription is activated.'
```

**Review by karelhala — COMMENTED** [review]
I wouldn't add the `package-lock.json` to this PR. Moreover it changes snapshot tests which are not directly conntected to this PR.

**Review by Hyperkid123 — APPROVED** [review]

File: src/messages/trial.messages.js (hunk 1 above)
Comment (ryelo) [reply src/messages/trial.messages.js]: Not all clouds urls go to console. Only the ones specified when we made the cutover. For example, this one shouldn't be redirecting.

Comment (aneelac22) [reply src/messages/trial.messages.js:168]: I reverted the changes I made

**Pull request merged**